DSCResources/MSFT_xPendingReboot/MSFT_xPendingReboot.schema.mof

```diff
@@ -2,10 +2,19 @@
 class MSFT_xPendingReboot : OMI_BaseResource
 {
   [key] string Name;
+  [write] boolean SkipComponentBasedServicing;
```

Comment:
**not blocking** I know the existing schema didn't have description, but it's a good practice to describe the parameters.